### PR TITLE
tools: replace upload_metrics

### PR DESCRIPTION
**Pull request by shiqizng**
adding a python script to print TPS from the block generator reports to stdout. the output is formatted in a way that can be copied and pasted to google sheets to create bar charts. 
And we decided not to use datadog for displaying the perf results; removing upload_metrics script. 

output: 
```
Scenario,Conduit_Version,Empty(no_reset)
payment.jumbo(25000),Conduit_Dev_Build_(May22),16982.46
asset.xfer(5000),Conduit_Dev_Build_(May22),10681.88
mixed(5000),Conduit_Dev_Build_(May22),11453.49
mixed.jumbo(25000),Conduit_Dev_Build_(May22),13390.13
payment.small(100),Conduit_Dev_Build_(May22),10010.13
payment.full(5000),Conduit_Dev_Build_(May22),16700.66
asset.destroy(5000),Conduit_Dev_Build_(May22),11481.67
asset.close(5000),Conduit_Dev_Build_(May22),15179.67
```

### CODE REVIEW

File: tools/block-generator/print_tps.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            tag, value = line.split(":")
+            data[tag] = value if tag == "scenario" else float(value)
+    return data
+
+
+def pretty_print(data):
+    table_header = (
+        f"Scenario,Conduit_Version,{args.database_description}"
+    )
+    print(table_header)
```
Comment (tzaffi) [inline tools/block-generator/print_tps.py:48]:
```suggestion
    print(f"Scenario,Conduit_Version,{args.database_description}")
```

ignoreable nit

File: tools/block-generator/print_tps.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+def parse_report(report):
+    data = dict()
+    with open(report) as f:
+        for line in f:
+            tag, value = line.split(":")
+            data[tag] = value if tag == "scenario" else float(value)
+    return data
+
+
+def pretty_print(data):
```
Comment (tzaffi) [inline tools/block-generator/print_tps.py:44]: I recommend a comment here explaining the conventions:
```python
"""
* scenario config files with "small" in their name are assumed to have `TXN_PER_BLOCK_SM` transactions per block
* scenario config files with "jumbo" in their names are assumed to have `TXN_PER_BLOCK_JUMBO` transactions per block
* others are assumed to have `TXN_PER_BLOCK` transactions per block
"""
```

File: tools/block-generator/print_tps.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    table_header = (
+        f"Scenario,Conduit_Version,{args.database_description}"
+    )
+    print(table_header)
+    for d in data:
+        scenario = d["scenario"].split("config.")[1]
+        scenario_parsed = scenario.split(".yml")[0]
+        txn_per_block = TXN_PER_BLOCK
```
Comment (tzaffi) [inline tools/block-generator/print_tps.py:52]: I don't think this is an issue, but just noting the fact that I'm about to change this exactness of these numbers going forward. For example, formerly the _small_ scenarios had **exactly** `TXN_PER_BLOCK_SM` (100) but now they may have $100 + \epsilon$ .

**Review by tzaffi — APPROVED** [review]
Approving as non of my proposed changes have any impact on function.

**Review by winder — APPROVED** [review]

**Pull request merged**